demo/universal/native/lib/ServerFunctions.re

```diff
@@ -216,7 +235,10 @@ module Notes = {
         )
       };
 
-    deleteHandler(~id) |> Lwt.map(response => React.Json(response));
+    //encode the response to json
```

Comment:
Can you remove those comments?

Reply:
Ops, it was present on the previous branch and I didn't pay enough attention on it. Fixing